apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -82,6 +82,17 @@ class ProjectCardGrid extends Component {
       <div id="projectCardGrid" style={styles.grid}>
         {this.state.showAll && (
           <div>
+            <ProjectAppTypeArea
+              labKey="special_topic"
+              labName={i18n.projectTypeSpecialTopic()}
+              labViewMoreString={i18n.projectTypeSpecialTopicViewMore()}
+              projectList={projectLists.special_topic}
+              numProjectsToShow={numProjects}
+              galleryType={this.props.galleryType}
+              navigateFunction={this.onSelectApp}
+              isDetailView={false}
+              hideWithoutThumbnails={true}
+            />
```

Comment:
Yay! This is coming along! 

Next steps:  
- [ ] add another, similar `ProjectAppTypeArea` in the `!this.state.showAll` section of this code
- [ ] add `special_topic` to the `projectLists` propType  in this component, like you did in `PublicGallery`
- [ ] put these changes behind an experiment flag